llvm/include/llvm/Frontend/OpenMP/OMP.td

```diff
@@ -2232,6 +2232,35 @@ def OMP_teams_loop : Directive<"teams loop"> {
   ];
   let leafConstructs = [OMP_Teams, OMP_loop];
 }
+def OMP_target_loop : Directive<"target loop"> {
```

Comment:
AFAIK, `target loop` is a non-standard construct, right? Where did we get the allowed clauses from? What `target loop` map to in the standard?

cc @mjklemm

Reply:
That is true. I copied the clauses from target and loop constructs.